puppet/environments/seed_stack/manifests/site.pp

```diff
@@ -46,22 +46,28 @@
 # Keep track of node IP addresses across the cluster
 # FIXME: A better, more automatic way to do this
 class seed_stack_cluster {
-  $controller_ip = '192.168.0.2'
-  $worker_ip = '192.168.0.3'
-
-  host { 'controller.seed-stack.local':
-    ip           => $controller_ip,
-    host_aliases => ['controller'],
-  }
-  host { 'worker.seed-stack.local':
-    ip           => $worker_ip,
-    host_aliases => ['worker'],
-  }
+  $controller_ip = '192.168.55.11'
+  $worker_ip = '192.168.55.21'
+
+  # The hostmanager vagrant plugin manages the hosts entries for us.
+
+  # host { 'controller.seed-stack.local':
```

Comment:
We can. I wasn't sure if I'd use `hostmanager` for this or something in puppet, and then I forgot to go back and delete them.